components/Post/LatestPosts.tsx

```diff
@@ -43,7 +43,9 @@ const LatestPosts = ({ latestPosts }: Props) => {
       ))}
 
       <Link href="/">
-        <a className="btn">See all posts</a>
+        <a className="btn" onClick={() => dispatch(setTags([]))}>
```

Comment:
![image](https://user-images.githubusercontent.com/46556479/195997417-0c4d4d84-f680-4055-a1a0-e4dc36650c78.png)

Works, but probbably makes more sense, to use a function, to reset it to default values

@ntorbinskiy

Reply:
@1alexvash ok